src/Akeneo/Connectivity/Connection/back/Infrastructure/Webhook/Command/SendBusinessEventToWebhooks.php

```diff
@@ -47,8 +52,18 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $message = \json_decode($input->getArgument('message'), true);
         $event = $this->bulkEventNormalizer->denormalize($message, BulkEvent::class);
 
-        $this->commandHandler->handle(new SendBusinessEventToWebhooksCommand($event));
-        $this->eventDispatcher->dispatch(new MessageProcessedEvent());
+        try {
+            $this->commandHandler->handle(new SendBusinessEventToWebhooksCommand($event));
+            $this->eventDispatcher->dispatch(new MessageProcessedEvent());
+        } catch (ConnectionException $ex) {
```

Comment:
```suggestion
        } catch (ConnectionException $exception) {
```
`$ex` don't talk to me, I prefer `$e` or `$exception`.
No big deal